libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -319,10 +319,19 @@ export class Function extends cloud.Function implements IAwsFunction {
       __dirname.replace("target-tf-aws", "shared-aws"),
       __filename,
       "FunctionClient",
-      [`process.env["${this.envName()}"], "${this.node.path}"`]
+      undefined,
+      this._liftedFields()
     );
   }
 
+  /** @internal */
+  public _liftedFields(): Record<string, string> {
```

Comment:
This API name honestly doesn't make sense, maybe it should be `_inflightNewArgs()` or something like that

Reply:
The thinking was: to create a client for resource r of type R you need some information from the preflight object r. This information is the unique information that isn't shared between all instances of R. Such information is typically (or always) stored in preflight fields of R (the env, ARN, special configs, etc).
In other words, what are the **fields** that are being lifted for all types R so they can be used inflight. These fields need to be lifted and stored (or used to initialized other inflight fields) in clients of R.

I feel `_inflightNewArgs` is missing what this essentially is, maybe `_liftedState()` makes more sense. We need this state to create a specific client for r.

Reply:
I see, yeah `_liftedState` may be better

Reply:
Updated